server/index.js

```diff
@@ -1,22 +1,32 @@
 import serverless from 'serverless-http';
 import express from 'express';
-import { getBits, getComments } from './src/store';
+import helmet from 'helmet';
+import { getBits, getComments, createBit } from './src/store';
 
 const app = express();
-
-app.get('/', (req, res) => {
-  res.send('Hello World!');
-});
+app.use(express.json()); // support encoded JSON
+app.use(express.urlencoded({ extended: true })); // support encoded bodies
+// Security middlewhere, but keep client-side caching.
+app.use(helmet({
+  noCache: {
+    action: 'deny'
+  }
+}));
 
 app.get('/bits', (req, res) => {
   res.setHeader('Content-Type', 'application/json');
   res.send(JSON.stringify(getBits()));
 });
 
+app.post('/bits', (req, res) => {
+  createBit(req.body.bit)
+      .then(result => res.location('/bits/' + result.postId).sendStatus(201))
+      .catch(error => res.status(500).send(error));

```

Comment:
Can you file an issue to clean up error handling? Outside of development we don't want to expose the error details.

Reply:
Done, #19